main/scripts/edit_panel.py

```diff
@@ -159,6 +161,8 @@ def create_edit_panel_widgets(self):
 
     def update_slider_value(self, event):
         current_option = self.edit_combobox.get()
+        if not hasattr(self, "current_image"):
```

Comment:
I couldn't find anywhere this attribute is properly initialized and couldn't initialize it in the class's __init__